data/po/tr.po

```diff
@@ -513,12 +501,8 @@ msgstr "{} bozuk. Lütfen yeniden indirin."
 #~ msgstr "Ayarlar"
 
 #~ msgctxt "show_fps_tooltip"
-#~ msgid ""
-#~ "Show the framerate while playing games. Only works with AMD and Nvidia "
-#~ "graphics cards."
-#~ msgstr ""
-#~ "Oyun oynarken kare hızını göster. Sadece AMD ve Nvidia ekran kartlarında "
-#~ "çalışır."
+#~ msgid "Show the framerate while playing games. Only works with AMD and Nvidia graphics cards."
```

Comment:
This part was changed, but it's still commented out. Is that intended?

Reply:
No, actually I didn't commented that part. Should i uncomment?

Reply:
Seems these are commented out everywhere. I think the string doesn't exist anymore, so it should be fine.